.github/workflows/main.yml

```diff
@@ -37,6 +37,7 @@ jobs:
 
       - name: Install/Link Postgres
         run: |
+          sudo apt update
```

Comment:
Alex added `sudo apt-get update -y -q` - not sure if the distinction matters here, but if the build ran/passed I'm sure it's fine.

Reply:
I added -yq in the style of the subsequent argument. (The online doc confirms those are both OK settings. I didn't have an interactive environment to try so we'll see when GA runs.)